canvas_api/src/types/responses/conversations.rs

```diff
@@ -0,0 +1,33 @@
+use chrono::{DateTime, Utc};
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct Conversation {
```

Comment:
The API looks kinda weird here, the Conversation returned when listing is different from that when getting.

It might be better to seperate these out to something like `ConversationOverview` and `Conversation`.